keyboards/kingly_keys/soap/rules.mk

```diff
@@ -0,0 +1,67 @@
+# MCU name
+MCU = atmega32u4
```

Comment:
Everything between `MCU` and `BOOTLOADER` can be removed, as it's set by default now.